### DISCUSSION TITLE
Updates and new things for the Electribe

### DISCUSSION THREAD

**Discussion by maxtribe** [bangcorrupt/hacktribe#125] [Q&A] [answered]
Hello guys

I have two questions. The first question would be about the execute file in thread #97. What I wanted to know is the following: 
Can this tool also be used for other things? Because I noticed that you can "only" modify the firmware of the Electribe with it.

And the second thing that I would be very interested in: When can we expect new updates?

Thank you for the information in advance

Warm greetings Max

**Comment by bangcorrupt** [comment, marked as answer]
The GUI can only be used to patch the firmware, other features will be added later.  Nothing will change for a while, I'm busy with other things.

**Comment by maxtribe** [comment]
Thank you for the answer
